wagtail/users/views/bulk_actions/assign_role.py

```diff
@@ -0,0 +1,73 @@
+from django import forms
+from django.contrib.auth.models import Group
+from django.template.response import TemplateResponse
+from django.utils.translation import gettext_lazy as _
+from django.utils.translation import ngettext
+
+from wagtail.core import hooks
+from wagtail.core.compat import AUTH_USER_APP_LABEL, AUTH_USER_MODEL_NAME
+from wagtail.users.views.bulk_actions.user_bulk_action import UserBulkAction
+
+
+change_user_perm = "{0}.change_{1}".format(AUTH_USER_APP_LABEL, AUTH_USER_MODEL_NAME.lower())
```

Comment:
You could import this from https://github.com/wagtail/wagtail/blob/main/wagtail/users/views/users.py#L29